### PR TITLE
bump dependencies and fix unit tests

### PR DESCRIPTION
**Pull request by zwk5004**
## Hey, I just made a Pull Request!

1. Unit tests are now working with this build
2. Backstage dependencies are bumped up to latest versions

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by Wisertown** [comment]
Tested locally and passed needed tests.

File: plugins/dashboard/src/hooks/useDashboard.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import useAsync from 'react-use/lib/useAsync';
 
 export function useDashboard() {
     const dashboardApi = useApi(dashboardApiRef);
+    console.log(dashboardApi);
```
Comment (Wisertown) [inline plugins/dashboard/src/hooks/useDashboard.ts]: do we need this console.log?

**Review by Wisertown — DISMISSED** [review]
Approved

**Review by sheppard-brandon-bah — APPROVED** [review]
LGTM. Build passed. Approving and merging.

**Pull request merged**